Guide/src/reference/openvmm/management/grpc.md

```diff
@@ -0,0 +1,15 @@
+# gRPC / ttrpc
```

Comment:
Having more docs on these interfaces would certainly be useful, though I think we'd want a bit more info on this page. Notably:

- a pointer to the .proto file which defines the interface
- mentioning the CLI flags that currently expose these APIs over the various transports
- a brief extra disclaimer summarizing the comment I left above, wrt. these APIs currently being at the POC stage, and that it is not clear whether we would go with a "external tool manages multiple OpenVMM instances" vs. "single OpenVMM instance manages multiple partitions" model